tx/textual/valuerenderer/timestamp_test.go

```diff
@@ -0,0 +1,164 @@
+package valuerenderer_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"cosmossdk.io/tx/textual/valuerenderer"
+	"github.com/stretchr/testify/require"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	dur "google.golang.org/protobuf/types/known/durationpb"
+	tspb "google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func TestTimestampRoundTrip(t *testing.T) {
```

Comment:
What does this test bring on top of `TestTimestampJsonTestcases`? It seems like the same test cases, and both are testing round trips.

Reply:
I developed this test first (the JSON wrangling was a pain), but at this point they cover the same ground. This one is more clear for Go maintainers, but redundancy has a cost too. Shall I delete this?

Reply:
+1 vote for delete this one

Reply:
Done. Added an "Error" field to the test cases and made more tests JSON-based.